### PR TITLE
perf: new ws listener option to disable UTF-8 validation

### PR DESCRIPTION
**Pull request by qzhuyan**
perf improvement for  #12344

Release version: v/e5.5

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx/src/emqx_schema.erl [hunk 1]
```diff
@@ -1103,6 +1103,14 @@ fields("ws_opts") ->
             sc(
                 ref("deflate_opts"),
                 #{}
+            )},
+        {"validate_utf8",
+            sc(
+                boolean(),
+                #{
+                    default => true,
```
Comment (zmstone) [inline apps/emqx/src/emqx_schema.erl:1111]: default to `false`?

**Comment by zmstone** [comment]
maybe change base to `master`.

File: apps/emqx/src/emqx_schema.erl (hunk 1 above)
Comment (qzhuyan) [reply apps/emqx/src/emqx_schema.erl:1111]: the default in cowboy is `true`, and I want to keep legacy behaviour. set it to false only when needed.

**Comment by qzhuyan** [comment]
note,
on a machine of 6 cores, apple m2. with reasonable socket buffer size, 
once I set it to `false`,  memory peak goes from 660MB to 470MB while testing with a 100M publish payload.

**Review by thalesmg — APPROVED** [review]

**Pull request merged**